src/Robo/Commands/Blt/DrushCliCommand.php

```diff
@@ -0,0 +1,163 @@
+      if (is_null($config_file)) {
+        $this->logger->warning("Could not find your CLI configuration file.");
+        $this->logger->warning("Looked in ~/.zsh, ~/.bash_profile, ~/.bashrc, ~/.profile, and ~/.functions.");
+        if (!$created) {
+          $this->logger->warning("Please create one of the aforementioned files, or create the Drush CLI aliases manually.");
+        }
+      }
+      else {
+          $source = $this->getConfigValue('repo.root');
+          $this->redispatchToVendorBin();
+          $this->setupComposerBinPlugin();
+          $this->createNewDrushCliAlias($source);
+      }
+    }
+
+    else {
+      $this->say("<info>The Drush CLI alias and dependencies are already installed.</info>");
+    }
+  }
+
+
+  /**
+   * Install and configure composer bin plugin. 
+   *
+   * @command setup:composer:bin-plugin
+   *
+   * @aliases composer-bin-plugin
+   */
+  public function setupComposerBinPlugin() {
+
+      $this->say('Adding composer vendor bin packages and config...');
+      $this->say('Adding drush 9 binaries and dependencies');
+       $result = $this->taskExec("composer bin drush-9 require drush/drush")
+          ->printOutput(TRUE)
+          ->dir($this->getConfigValue('repo.root'))
+          ->run();
+
+      $this->say('Adding drush 8 binaries');
+       $result = $this->taskExec("composer bin drush-8 require drush/drush:8.1.16")
+          ->printOutput(TRUE)
+          ->dir($this->getConfigValue('repo.root'))
+          ->run();
+
+      $this->say('Adding drush 8 wrapper');
+      //$source_path = $this->getConfigValue('blt.root') . '/templates/drush/drush' ;
+      //$target_path = $this->getConfigValue('repo.root') . "/vendor-bin/drush-8/drush" ;
+
+      $result = $this->taskFilesystemStack()
+        ->copy($this->getConfigValue('blt.root') . '/template/drush/drush', $this->getConfigValue('repo.root') . '/vendor-bin/drush-8/drush', TRUE)
+        ->stopOnFail()
+        ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+        ->run();
+
+      if (!$result->wasSuccessful()) {
+        throw new BltException("Could not copy drush wrapper.");
+      }
+
+     /* // Fail silently if target file already exists.
+      if (!$this->getFileSystem()->exists($target_path)) {
+        $this->getFileSystem()->rename($source_path, $target_path);
+        $this->say("Drush 8 wrapper created");
+      }
+       else {
+          $this->logger->warning("Error creating drush 8 wrapper. Create manually and/or review documentation via drush8 topic core-global-options.");
+        }*/
+
+      $this->say("<info>The Drush CLI alias and dependencies are already installed.</info>");
+    }
+
+
+  /**
+   * Prevent re-dispatch to site local drush bin in favor of vendor-bin to 
+   * support running legacy Drush 8 commands. 
+   *
+   * @command blt:drush:redispatch
+   *
+   * @aliases redispatch
+   */
+
+  public function redispatchToVendorBin() {
```

Comment:
This seems like it just remove Drush 9 from vendor. Would it be more appropriately named `removeDrush9()` ?